geometry_msgs/Point robotPos
---
geometry_msgs/Point point

